packages/tldraw/src/lib/shapes/shared/PlainTextLabel.tsx

```diff
@@ -107,7 +110,12 @@ export const PlainTextLabel = React.memo(function PlainTextLabel({
 					height: textHeight ? Math.ceil(textHeight) : undefined,
 				}}
 			>
-				<div className={`${cssPrefix} tl-text tl-text-content`} dir="auto">
+				<div
+					className={classNames(`${cssPrefix} tl-text tl-text-content`, {
+						'tl-text__no-outline': !showTextOutline,
```

Comment:
i think this also needs the `showTextOutline ? 'tl-text__outline' : 'tl-text__no-outline'` logic like in `RichTextLabel` unless i'm missing something

Reply:
good! catch